### ISSUE TITLE
The virtual machine 'metasploitable3-win2k8' has terminated unexpectedly during startup with exit code 1 (0x1)

### ISSUE THREAD

**Issue by jdepalma** [rapid7/metasploitable3#273]
## Host System
* OS: Windows 10 Home.
* Packer Version: 1.2.2 (Also tried 1.0.0) 
* Vagrant Version: 1.9.2 (Also tried 1.9.1)
* VirtualBox Version: 5.1.14 (As suggested)

## Command Output

Compatible version of VirtualBox found.
Compatible version of packer found.
Compatible version of Vagrant found.
Compatible version of vagrant-reload plugin found.
All requirements found. Proceeding...
Building the Vagrant box...
virtualbox-iso output will be in this color.

==> virtualbox-iso: Downloading or copying Guest additions
    virtualbox-iso: Downloading or copying: file:///C:/Program%20Files/Oracle/VirtualBox/VBoxGuestAdditions.iso
==> virtualbox-iso: Downloading or copying ISO
    virtualbox-iso: Downloading or copying: http://download.microsoft.com/download/7/5/E/75EC4E54-5B02-42D6-8879-D8D3A25FBEF7/7601.17514.101119-1850_x64fre_server_eval_en-us-GRMSXEVAL_EN_DVD.iso
==> virtualbox-iso: Creating floppy disk...
    virtualbox-iso: Copying files flatly from floppy_files
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../answer_files/2008_r2/Autounattend.xml
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../scripts/configs/microsoft-updates.bat
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../scripts/configs/win-updates.ps1
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../scripts/installs/openssh.ps1
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../scripts/installs/install_dotnet45.ps1
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../scripts/installs/install_wmf.ps1
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/oracle-cert.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/gdig2.crt
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/comodorsadomainvalidationsecureserverca.crt
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/comodorsacertificationauthority.crt
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/addtrust_external_ca.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/baltimore_ca.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/digicert.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/equifax.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/globalsign.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/gte_cybertrust.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/microsoft_root_2011.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/thawte_primary_root.cer
    virtualbox-iso: Copying file: F:\Users\shinj\metasploitable3-master\packer\templates/../../resources/certs/utn-userfirst.cer
    virtualbox-iso: Done copying files from floppy_files
    virtualbox-iso: Collecting paths from floppy_dirs
    virtualbox-iso: Resulting paths from floppy_dirs : []
    virtualbox-iso: Done copying paths from floppy_dirs
==> virtualbox-iso: Creating virtual machine...
==> virtualbox-iso: Creating hard drive...
==> virtualbox-iso: Attaching floppy disk...
==> virtualbox-iso: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 2664)
==> virtualbox-iso: Executing custom VBoxManage commands...
    virtualbox-iso: Executing: modifyvm metasploitable3-win2k8 --memory 4096
    virtualbox-iso: Executing: modifyvm metasploitable3-win2k8 --cpus 2
==> virtualbox-iso: Starting the virtual machine...
==> virtualbox-iso: Error starting VM: VBoxManage error: VBoxManage.exe: error: The virtual machine 'metasploitable3-win2k8' has terminated unexpectedly during startup with exit code 1 (0x1).  More details may be available in 'C:\Users\shinj\VirtualBox VMs\metasploitable3-win2k8\Logs\VBoxHardening.log'
==> virtualbox-iso: VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component MachineWrap, interface IMachine
==> virtualbox-iso: Unregistering and deleting virtual machine...
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Error starting VM: VBoxManage error: VBoxManage.exe: error: The virtual machine 'metasploitable3-win2k8' has terminated unexpectedly during startup with exit code 1 (0x1).  More details may be available in 'C:\Users\shinj\VirtualBox VMs\metasploitable3-win2k8\Logs\VBoxHardening.log'
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component MachineWrap, interface IMachine

==> Some builds didn't complete successfully and had errors:
--> virtualbox-iso: Error starting VM: VBoxManage error: VBoxManage.exe: error: The virtual machine 'metasploitable3-win2k8' has terminated unexpectedly during startup with exit code 1 (0x1).  More details may be available in 'C:\Users\shinj\VirtualBox VMs\metasploitable3-win2k8\Logs\VBoxHardening.log'
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component MachineWrap, interface IMachine

==> Builds finished but no artifacts were created.

**Comment by jdepalma** [comment]
I have made every attempt to ensure AV was out of the picture by disabling Defender, my AV, and Firewall. I've also tried to build manually and have downloaded manually the windows ISO and placed it in the ISO folder (the script still downloaded the same ISO though..no Idea where it stores that). I have attempted to use different versions of packer and vagrant.

The issue appears to be with something with metasploitable3-win2k8. Earlier I had problems with metasploitable3-1404 or some such.
